EFAuditer/EFAuditServiceExtensions.cs

```diff
@@ -0,0 +1,33 @@
+using System;
+using System.Linq;
+using Audit.Core;
+using EFAuditer;
+using Microsoft.EntityFrameworkCore;
+
+namespace Microsoft.Extensions.DependencyInjection
+{
+    public static class EFAuditServiceExtensions
+    {
+        public static void AddEFAuditer(this IServiceCollection services, string connectionString)

```

Comment:
This is following the convention for adding services, which should all live in the DependencyInjection namespace and extend IServiceCollection :)

Reply:
Looks good!